modules/ducktests/tests/ignitetest/tests/thin_client_test.py

```diff
@@ -32,6 +32,11 @@ class ThinClientTest(IgniteTest):
     """
     cluster - cluster size.
     JAVA_CLIENT_CLASS_NAME - running classname.
+    to use with ssl enabled:
+    export GLOBALS='{"ssl":{"enabled":true},"key_store_path":"/opt/ignite-dev/modules/ducktests/tests/certs/",
```

Comment:
Actually, we don't need anything except "ssl:enabled:true".
Please, fix the doc.